tests/foreman/ui_airgun/test_discoveryrule.py

```diff
@@ -0,0 +1,103 @@
+# -*- encoding: utf-8 -*-
+"""Test class for Foreman Discovery Rules
+
+:Requirement: Discoveryrule
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: UI
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+
+from fauxfactory import gen_integer, gen_string
+from nailgun import entities
+
+
+def test_positive_create(session):
+    name = gen_string('alpha')
+    search = '{}\t'.format(gen_string('alpha'))
+    org = entities.Organization().create()
+    host_group_name = gen_string('alpha')
+    with session:
+        session.organization.select(org_name=org.name)
+        session.hostgroup.create({'name': host_group_name})
```

Comment:
I think that is good enough to create that entity through API `hg = entities.HostGroup().create()` as it is completely out of scope for that module

Reply:
Yes I understand and i don't wanna to create it , but it have a purpose why i did is this way because i was not able to specify the organization in these nailgun entities, I specify more question about it in https://github.com/SatelliteQE/airgun/pull/152

Reply:
in that way, it should work. Otherwise, provide exact exception that you get in debug